src/Symfony/Component/DependencyInjection/Tests/Compiler/MergeExtensionConfigurationPassTest.php

```diff
@@ -6,6 +6,8 @@
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\ParameterBag\ParameterBag;
 
+require_once __DIR__.'/../Fixtures/includes/PrependExtension.php';
```

Comment:
Can do, but do you have any preference? I put it there as `ContainerBuilderTest` does a similar thing with a `ProjectExtension` class.

Would `Tests/Compiler` be a good location for this (i.e. same directory as the test)?